backend/__tests__/config.spec.cjs

```diff
@@ -167,14 +122,44 @@ describe('config', function () {
           throw new Error(filePath + ': not found')
         })
 
-        const filename = '/etc/gardener/4/config.yaml'
+        const filename = '/etc/gardener/config.yaml'
         const config = gardener.loadConfig(filename, { env })
 
         expect(config.oidc.issuer).toBe('https://kubernetes:32001')
         expect(config.oidc.client_id).toBe('client_id_from_file')
         expect(config.oidc.client_secret).toBe('client_secret_from_file')
         expect(config.gitHub.authentication.appId).toBe(12345)
         expect(config.gitHub.authentication.installationId).toBe(67890)
+        expect(config.websocketAllowedOrigins).toEqual(['*'])
+        expect(config.logLevel).toBe('info')
+      })
+
+      it('should return the config overridden by environment variables', function () {
```

Comment:
add test for empty websocketAllowedOrigins

Reply:
I removed the variables completely from the test.
The empty array case is tested in `io.cors.spec.cjs` *should reject connections if no origins are configure*